pygaggle/data/msmarco.py

```diff
@@ -131,14 +129,11 @@ def to_relevance_examples(self,
             mean_stats['Existing MRR'].append(1 / (ex_index + 1)
                                               if ex_index < len(ex.candidates)
                                               else 0)
-            mean_stats['Existing MRR@10'].append(1 / (ex_index + 1)
-                                                 if ex_index < 10 else 0)
+            mean_stats['Existing MRR@10'].append(1 / (ex_index + 1) if ex_index < 10 else 0)
```

Comment:
Won't switching make flake8 angry? Unless we plan to go with a different line length than 80, in which case this is fine!

Reply:
Yeah, we've decided to go with 100.